src/lib/markbind/src/handlers/cyclicReferenceError.js

```diff
@@ -0,0 +1,20 @@
+const MAX_RECURSIVE_DEPTH = 200;
+const ERROR_MESSAGE = 'Cyclic reference detected.';
+
+class CyclicReferenceError extends Error {
+  static get MAX_RECURSIVE_DEPTH() {
+    return MAX_RECURSIVE_DEPTH;
```

Comment:
Personally i feel that writing a static getter is the same as writing a constant. We could remove the constant above and update the getter as:

```suggestion
    return 200;
```